sda/cmd/auth/main.go

```diff
@@ -143,7 +143,7 @@ func (auth AuthHandler) postEGA(ctx iris.Context) {
 		if ok {
 			log.WithFields(log.Fields{"authType": "cega", "user": username}).Info("Valid password entered by user")
 			claims := map[string]interface{}{
-				jwt.ExpirationKey: time.Now().UTC().Add(200 * time.Hour),
+				jwt.ExpirationKey: time.Now().UTC().Add(time.Duration(auth.Config.JwtTTL) * time.Hour),
```

Comment:
The adding part here could be without the `time.Duration` I think. it would be the same and cleaner since the `auth.Config.JwtTTL` is an integer. So it could just be like:
`jwt.ExpirationKey: time.Now().UTC().Add(auth.Config.JwtTTL * time.Hour),`

Reply:
No you can't, if you remove the `time.Duration` you get compiler error because of type mismatch.
Basically you can't multiply two variables of different types. https://go.dev/play/p/ERG5fh1N0Sf